repos/cli.proj

```diff
@@ -39,6 +39,9 @@
     <RepoApiImplemented>false</RepoApiImplemented>
     <SourceOverrideRepoApiImplemented>true</SourceOverrideRepoApiImplemented>
 
+    <!-- Temporarily disable auto-update: patch consistently conflicts with dependency changes. -->
```

Comment:
Do we need to re-disable this?

Reply:
Oh, this was unintentional, must have messed up my rebase flow. Fixing.